packages/firestore/src/core/view.ts

```diff
@@ -275,13 +275,16 @@ export class View {
    *        this change.
    * @param targetChange - A target change to apply for computing limbo docs and
    *        sync state.
+   * @param waitForRequeryResult - Whether the target is pending to run a full
+   *        re-query due to existence filter mismatch.
    * @returns A new ViewChange with the given docs, changes, and sync state.
    */
   // PORTING NOTE: The iOS/Android clients always compute limbo document changes.
   applyChanges(
     docChanges: ViewDocumentChanges,
     updateLimboDocuments: boolean,
```

Comment:
I know it's not technically part of your change, but consider renaming the "updateLimboDocuments" parameter of `applyChanges()`. Previously, this parameter exclusively controlled whether limbo resolution would happen, if needed; however, with the introduction of the "waitForRequeryResult" parameter, specifying `updateLimboDocuments=true` does not necessarily cause limbo resolution to occur. Consider renaming the parameter to something like "limboResolutionEnabled" because IMO such a name more clearly conveys that limbo resolution may not occur if it's not needed.